src/app/clientPage.tsx

```diff
@@ -93,7 +98,8 @@ export default function ClientPage({ searchParams }: { searchParams?: { [key: st
     return <Loading reset={reset} summaryContent={originalContent} songDetails={songDetails} />;
 
   return (
-    <>
+    // Subtracting footer height from 100vh
+    <div className="min-h-[calc(100vh-7.9rem)] md:min-h-[calc(100vh-5rem)]">
```

Comment:
Perhaps we want to keep a reference to the height in a variable so we can reuse?